test/performance/performance-tests-mako.sh

```diff
@@ -192,11 +230,11 @@ for i in {1..60}; do
 done
 echo ""
 
-###############################################################################################
+##############################################################################################
```

Comment:
guessing the commented out lines are tests we're not currently running?

Reply:
yeah, will start with a few tests and add more over time